src/components/EditorHeader/ControlPanel.jsx

```diff
@@ -1374,15 +1376,20 @@ export default function ControlPanel({
 
   function toolbar() {
     return (
-      <div className="py-1.5 px-5 flex justify-between items-center rounded-xl my-1 sm:mx-1 xl:mx-6 select-none overflow-hidden toolbar-theme">
+      <div
+        className="py-1.5 px-5 flex justify-between items-center rounded-xl my-1 sm:mx-1 xl:mx-6 select-none overflow-hidden toolbar-theme"
+        style={isRtl(i18n.language) ? { direction: "rtl" } : {}}
+      >
         <div className="flex justify-start items-center">
           <LayoutDropdown />
           <Divider layout="vertical" margin="8px" />
           <Dropdown
             style={{ width: "240px" }}
             position="bottomLeft"
```

Comment:
Could you please add this to `position` just make the dropdowns all consistent

```js
isRtl(i18n.language) ? "bottomRight" : "bottomLeft"
```

Reply:
@1ilit Thanks so much for the code review, I've learned things! Please check my latest commit. I have addressed the issues you mentioned. Also, I made both field name and data type ltr as they will always be in English.
![image](https://github.com/user-attachments/assets/796690a8-e8c8-4fca-9135-b6e0efd8bc91)